### DISCUSSION TITLE
Advice for First setup

### DISCUSSION THREAD

**Discussion by robons17** [awawa-dev/HyperHDR#489] [Q&A]
Hi,
First of all thanks for this job.
I'm Building my First setup. Buy i have a questione : how manager dolby visione content? I'm planning to go with ezcoo splitter - rullz 2019 - rpi4 - esp32 ( doubt between serial or spi) .
I think It Is not ok for dv. 

Do i Need a new capture card ?
Does this setup look ok?

Thanks.

**Comment by awawa-dev** [comment]
Hi
DV is closed proprietary system and I don't know any solution that can capture it properly. Choose HyperSerialESP32 if your ESP board has a serial chip that is capable of high speed (>= 2Mb speed is recommended but can be  easily compiled online also for 1Mb, still it should be fine if don't use LED strip multi-segments). If you experience problems with your USB2.0 grabber while using it (reduced capture framerate) switch to HyperSPI...even in such situation I think it can be workaround using MJPEG format which is friendly for USB2.0 bus. USB3.0 grabbers are not affected but all MS2109 variants are true USB2.0 grabbers.

**Reply by robons17** [reply]
I have some doubt on framerate. I have ti test grabber at the end of this month.

**Reply by awawa-dev** [reply]
I think it should be fine. In the HyperSPI example you have a Ezcap 320 grabber which uses very high uncompressed resolution at very high framerate so it saturates the USB2.0 bus: MS2109 can't do the same thing, because it offers only MJPEG compressed format for higher resolution or higher resolution anyway.

**Reply by robons17** [reply]
I ordered a ms2109 ( rullz). The main doubt Is change It tò use uncompressed format with high framerate.  As resolution i think to use 720p or 480p....

**Reply by awawa-dev** [reply]
https://www.hyperhdr.eu/2020/11/grabber-for-hyperhdr-rullz-usb-20.html
For uncompressed 720p MS2109 offers only 10fps so that mode is useless.
Capturing rate has much higher priority for smooth ambient lighting than resolution.

**Reply by robons17** [reply]
Hi awawa-dev,
I start design the frame for led.
![image](https://user-images.githubusercontent.com/22059192/213936460-4e404ac9-c8e7-4b0f-9baa-fa3ad5848909.png)
What do you think about it? (45 degrees towards wall--> I read it that is the suggested config up to now). 
The outer size is the same of the TV (1236*710).
Thanks.

**Reply by awawa-dev** [reply]
The angle depends on how far the LED strip is from the wall. I have 45 angle on my setup but the frame is very close to the wall (~3cm) . The frame should be rather a bit smaller than TV...al least I wanted it to be hidden behind TV while watching movies.

**Reply by robons17** [reply]
In my setup i think i can place the brame from 6 to 12 cm from the wall. More or less the same distance between frame and tv.

**Reply by awawa-dev** [reply]
Then you may want to consider a smaller angle, 30° or less, as at 12 cm the maximum light intensity will be quite far from the TV for 45°

**Reply by robons17** [reply]
In your opinion waht is the distance to have 90° between wall and tv? (60led/meter) . 
Thanks

**Reply by awawa-dev** [reply]
I think I dont quite understand such setup. Anyway LED intensity drops significantly above 30° (this is not the mounting angle, but the light coming out from the LED itself) so if you know the distance LED strip to the wall you can calculate where such zone for 30° starts for given mounting angle.
![obraz](https://user-images.githubusercontent.com/69086569/213939142-3d83b815-771a-44ac-8982-b4637a71e2ac.png)

**Comment by robons17** [comment]
Hi [awawa-dev],
thanks for the advice, the design is work in progress, but can i ask your opinion about the box and the layout?
![image](https://user-images.githubusercontent.com/22059192/215900475-8fb9f18f-2524-4eeb-a892-9b0868cd6c6a.png)
I have to found a 5 pin panel conn. for led output...

**Reply by awawa-dev** [reply]
Do you plan separate USB power supply for Rpi. Why 5 pin panel connector for LEDs?

**Reply by robons17** [reply]
The t shape object in near the AC compartment rappresent rpi Power supply.
The 5 PIN Is for have 3 STD wire and 2 for inject 5v.

Dimensione are 25x23x5 cm.
I'm thinking tò mount 40x40 fan on top of rpi.

**Reply by awawa-dev** [reply]
I would worry more about the temperature rise caused by the LED power supply (check its specifications, including efficiency, the parameters change with the increase in temperature) than about the Rpi that will be slightly loaded. At least passive cooling (e.g. ventilation holes) must be unconditionally provided.

**Reply by ghost** [reply]
It would be nice to use a quality power supply from a good manufacturer (Mean Well), with a small margin of power.

**Comment by ghost** [comment]
> I'm planning to go with ezcoo splitter - rullz 2019 - rpi4 - esp32 ( doubt between serial or spi) .

I have a similar set that I am currently buying for assembly.
But I'm wondering if it's worth buying a new MS2130 capture card instead of a MS2109.
There are not so many reviews on the new card, I did not find their direct comparison (color reproduction, delay).
The reviews have comments on color reproduction, although according to some comments this seems to have been fixed by updating the firmware. About the delay they write about 50-100 ms.
<details>
  <summary>Couple of links</summary>
  
  https://www.bilibili.com/video/BV1CG411K7DC
  https://twitter.com/MateuszStarzak/status/1588480134276939777

  ![Screenshot_1](https://user-images.githubusercontent.com/85223482/216290055-08ff3cfe-206b-4036-ae19-e2f486b19a80.png)

</details>

**Reply by awawa-dev** [reply]
HyperHDR has built-in benchmarks (latency and automatic LUT calibration/HDR quality) and I will test MS2130 in 3 or 4 weeks. Old MS2109 has delay of 70-80ms. Hope they didn't break colors in these new grabbers, because I don't think firmware upgrade will be possible for them.

**Comment by awawa-dev** [comment]
@pavel-ponomarev I've just received MS2130. So far it looks very good. Beta dedicated LUT table for this grabber is already available: https://github.com/awawa-dev/HyperHDR/discussions/499

**Reply by ghost** [reply]
Good afternoon. I have already ordered this capture card (only option A, with a rounded body).
Pending delivery)

**Comment by robons17** [comment]
Hi awawa-dev,
I have almost completed the box that will contain everything.
Just two last questions:

- what cable size do you recommend for 5v and led data line?

I  go with multi-segment for my setup. I think however that using a 4 pin molex connector (5v + data line) is not enough. (Molex supports 11A per pin).
What do you advise?  An alternative would be to use a 4 pin molex for the two 5v lines and a 2 pin jst for the two data lines.

Thank you,

R

**Reply by awawa-dev** [reply]
Hi
On the blog there is an answer for your question: xt60 (and molex just for the data line). It's even visible on my final picture of my box: the yellow plug on the left..I don't have to use it anyway, my 300RGBW consumes only a maximum of ~35W after measurements, but this is my configuration characteristics and you have to measure the power consumption of your configuration yourself.
